pkg/porter/build.go

```diff
@@ -338,6 +339,33 @@ func (p Porter) writeBundle(b bundle.Bundle) error {
 	return errors.Wrap(err, "error writing to cnab/bundle.json")
 }
 
+func (p *Porter) generateBundleImages() map[string]bundle.Image {
+	images := make(map[string]bundle.Image, len(p.Manifest.ImageMap))
+
+	for i, refImage := range p.Manifest.ImageMap {
+		img := bundle.Image{
+			Description: refImage.Description,
+			BaseImage: bundle.BaseImage{
+				Image:         refImage.Image,
+				Digest:        refImage.Digest,
+				ImageType:     refImage.ImageType,
+				MediaType:     refImage.MediaType,
+				OriginalImage: refImage.OriginalImage,
+				Size:          refImage.Size,
+			},
+		}
+		if refImage.Platform != nil {
```

Comment:
Do we want to add a quick unit test around a mappedImage that doesn't have `Platform` defined?